package/MDAnalysis/analysis/msd.py

```diff
@@ -382,16 +409,30 @@ def _conclude(self):
             self._conclude_simple()
 
     def _conclude_simple(self):
-        r""" Calculates the MSD via the simple "windowed" algorithm.
+        r""" Calculates the MSD via the simple "windowed" algorithm 
+        with nongaussian parameter.
 
         """
         lagtimes = np.arange(1, self.n_frames)
         positions = self._position_array.astype(np.float64)
         for lag in lagtimes:
             disp = positions[:-lag, :, :] - positions[lag:, :, :]
             sqdist = np.square(disp).sum(axis=-1)
-            self.results.msds_by_particle[lag, :] = np.mean(sqdist, axis=0)
+            tmp = np.mean(sqdist, axis=0)
```

Comment:
Why the need to use a temporary?

Reply:
Mostly for initial debugging, it's gone now.